### PR TITLE
Adding a reference architecture template

### PR DESCRIPTION
**Pull request by haleycode**

### CODE REVIEW

File: static/_templates/_reference-architecture.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: xx
+pcx_content_type: reference-architecture
+---
+ 
+<!---
+Cloudflare employees should refer to the "Guidelines for creating architectures"  page of the PCX wiki for more information on information you are required to include in reference architecture documents, formatting information, and Markdown components.
```
Comment (crwaters16) [inline static/_templates/_reference-architecture.md]: Small tweaks:
"...page of the PCX wiki for more information on what content you are required to include in reference architecture documents, including formatting information and Markdown components."

File: static/_templates/_reference-architecture.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+-->
+
+# <PRODUCT> Reference architecture
```
Comment (crwaters16) [inline static/_templates/_reference-architecture.md]: "reference" will be lowercased following the product name

File: static/_templates/_reference-architecture.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+ 
+## Brief introduction title
+ 
+<!--- In this section you should provide a brief 1-2 paragraph introduction to the reference architecture document covering who the documentation will serve, which products are involved and the scope of the document.
```
Comment (crwaters16) [inline static/_templates/_reference-architecture.md]: We serial comma in this operation! (...involved, and the scope...)

**Review by crwaters16 — APPROVED** [review]

File: static/_templates/_reference-architecture.md (hunk 1 above)
Comment (haleycode) [reply static/_templates/_reference-architecture.md]: But what if I added the word "information" three MORE times? XD Thanks!

**Pull request merged**